README.md

```diff
@@ -17,7 +17,7 @@ _Service for storing relations between foreign id (resources)._
 
    ```bash
    git clone https://github.com/Sundsvallskommun/api-service-relations.git
-   cd api-service-relations
+   cd api-RelationService-relations
```

Comment:
Typo? It should be api-service-relations, right?

Reply:
Refactor (search and replace) misstake.